docs/addons/neuron-plugin-addon.md

```diff
@@ -0,0 +1,53 @@
+# Neuron Device Plugin Addon
+
+[AWS Neuron](https://awsdocs-neuron.readthedocs-hosted.com/en/latest/) is the SDK used to run deep learning workloads on AWS Inferentia and AWS Trainium based instances. This addon will install the Neuron Device Plugin necessary to run the instances on Amazon EKS (and Blueprints). Note that you **must** use a GPU nodegroup with *inf1, inf2, trn1,* or *trn1n* instances.
```

Comment:
I assume we can call out that it does not have to be a node group, could be just a karpenter provisioner, is that correct?

Reply:
addressed.